content/zh/docs/reference/using-api/api-concepts.md

```diff
@@ -1350,11 +1347,7 @@ This table explains the behavior of **list** requests with various combinations
 | `resourceVersionMatch=NotOlderThan` | _limit unset_                 | Invalid               | Any                                       | Not older than                         |
 | `resourceVersionMatch=NotOlderThan` | limit=\<n\>, _continue unset_ | Invalid               | Any                                       | Not older than                         |
 
-{{</* /table */>}}
-
-**Footnotes:**
-
-[1] If the server does not honor the `resourceVersionMatch` parameter, it is treated as if it is unset.
+{{< /table >}}
```

Comment:
This is the line that causes the problem. Please change it to:

```
{{</* /table */>}}
```

Reply:
This line must be changed, don't you understand? It has broken the whole website CI gate, and it has caused the PR to be reverted.